binsync/decompilers/ghidra/controller.py

```diff
@@ -130,8 +130,40 @@ def fill_function_header(self, func_addr, user=None, artifact=None, decompilatio
     @fill_event
     @ghidra_transaction
     def fill_struct(self, struct_name, header=True, members=True, artifact=None, **kwargs):
-        struct: Struct = artifact;
-
+        changes = True
+        struct: Struct = artifact
+        old_ghidra_struct = self._get_struct_by_name('/'+struct_name)
+        data_manager = self.ghidra.currentProgram.getDataTypeManager()
+        t = self.ghidra.currentProgram.startTransaction('add struct')
+        try:
+            handler = self.ghidra.import_module_object("ghidra.program.model.data", "DataTypeConflictHandler")
+            structType = self.ghidra.import_module_object("ghidra.program.model.data", "StructureDataType")
+            byteType = self.ghidra.import_module_object("ghidra.program.model.data", "ByteDataType")
+            ghidra_struct = structType(struct_name, 0)
+            for offset in struct.members:
+                member = struct.members[offset]
+                ghidra_struct.add(byteType.dataType, 1, member.name, "")
+                ghidra_struct.growStructure(member.size-1)
+                for dtc in ghidra_struct.getComponents():
+                    if dtc.getFieldName() == member.name:
+                        if member.type:
```

Comment:
Any time you find yourself doing a single assignment or call based on 1 or 2 conditions, you should use inline conditional checks to make it Pythonic: 
```suggestion
                        gtype = self.typestr_to_gtype(member.type if member.type else 'undefined'+str(member.size))
```

Reply:
(this code should replace the entire if-stmt tree)